x-pack/test_utils/jest/contract_tests/default_server_config.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resolve } from 'path';
+
+export const xpackOption = {
```

Comment:
This config feels like it's pretty specific to a specific set of tests, not all contract tests that we'll likely write in the near term. I can also imagine that changing this config will break/impact existing contract tests, so i think it'd be better to put this local to the current tests rather than in the shared location.